src/KeyFrame.cc

```diff
@@ -926,6 +1022,37 @@ double MultiKeyFrame::Distance(MultiKeyFrame &other)
   return dMinDist;
 }
 
+//partially copies MKF
+MultiKeyFrame* MultiKeyFrame::CopyMultiKeyFramePartial()
```

Comment:
The Copy...Partial is kind of an unclear name.  What part is it copying or for what purpose are you making a partial copy? Is it speed or memory concerns instead of just storing the whole MKF in the buffer?